client/blocks/app-promo/index.jsx

```diff
@@ -148,29 +153,36 @@ export class AppPromo extends Component {
 		const { translate } = this.props;
 		const buttons = [ { action: 'cancel', label: translate( 'OK' ) } ];
 
+		const message = displayJetpackAppBranding ? (
+			<p className="app-promo__paragraph">
+				<a href="https://wp.com/app" className="app-promo__jetpack-app-link">
+					Get the Jetpack app
+				</a>{ ' ' }
+				{ translate( 'to use Reader anywhere, any time.' ) }
+			</p>
+		) : (
+			<p className="app-promo__paragraph">
```

Comment:
Seems like in both cases we render a `<p className="app-promo__paragraph" />`. Should the conditional logic be inside the `<p>` then?

Reply:
Ah, good point! I've made that change in https://github.com/Automattic/wp-calypso/pull/68293/commits/edf541644a1c27bf97da13f594cccf52a40fda77.